README.md

```diff
@@ -23,113 +23,108 @@ _Prerequisites:_ Chunky requires **Java 8**. It is recommended to have the **64-
 
 If you have any questions, please don't hesitate to reach out via Reddit, Discord or GitHub.
 
+
 ## Frequently Asked Questions
 
 <details>
 <summary><strong>Why is there noise/grain/random bright dots in the render?</strong></summary>
 
-This is not a bug, but an unfortunate effect of [the rendering algorithm][9] used in Chunky. Torches and other small light sources cause a very random illumination and it takes a long time to render such light nicely.
-
-
-You can disable emitters under the Lighting tab in the Render Controls dialog to remove most of the random bright dots. Note that rendering for a longer time will eventually remove the noise, though it may take a very long time.
-
-Another way of removing the noise is using the [Denoiser Plugin][10]. While this can yield good results in most cases, it may distort the image in some cases.
+> This is not a bug, but an unfortunate effect of [the rendering algorithm][9] used in Chunky. Torches and other small light sources cause a very random illumination and it takes a long time to render such light nicely.
+> 
+> You can disable emitters under the Lighting tab in the Render Controls dialog to remove most of the random bright dots. Note that rendering for a longer time will eventually remove the noise, though it may take a very long time.
+> 
+> Another way of removing the noise is using the [Denoiser Plugin][10]. While this can yield good results in most cases, it may distort the image in some cases.

```

Comment:
Why are the answers quotes now?

Reply:
Before
![image](https://user-images.githubusercontent.com/3740917/108778578-77f6bf80-755d-11eb-9b1b-fba2bb568d73.png)

After
![image](https://user-images.githubusercontent.com/3740917/108778596-7f1dcd80-755d-11eb-822c-9683895db80e.png)

Maybe quotes aren't the best formatting option to use but it does help improve readability by breaking things up a bit more. It was a ninja change which looked good.